tests/tasks/setup/eks/awscli-cp.yaml

```diff
@@ -28,8 +28,8 @@ spec:
     description: |
       A workspace into which a kubeconfig file called `kubeconfig` will be written that will contain the information required to access the cluster. The `kubeconfig` will expect to use [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator/) to authenticate, so in order for it to be used it must be run in a container which contains both `kubectl` and `aws-iam-authenticator`.
   steps:
-  - name: write-kubeconfig
-    image: alpine/k8s:1.23.13
+  - name: create-cluster
+    image: alpine/k8s:1.23.7
```

Comment:
looks like you might have picked up file from older commit,  please update the version back to new oner.

Reply:
as discussed offline done this to make the images consistent.